README.md

```diff
@@ -128,17 +128,21 @@ end
 `Fail2Ban.filter` can be used within a blacklist to block all requests from misbehaving clients.
 This pattern is inspired by [fail2ban](http://www.fail2ban.org/wiki/index.php/Main_Page).
 See the [fail2ban documentation](http://www.fail2ban.org/wiki/index.php/MANUAL_0_8#Jail_Options) for more details on
-how the parameters work.
+how the parameters work.  All fail2ban filters share the same match counter, using one filter per app is recommended.
```

Comment:
@jamesprior Can you clarify what you mean by `All fail2ban filters share the same match counter`?